test/data/raml10-oas30/target/baseUri.yaml

```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.0
 servers:
-  - url: 'http://93.144.28.153:8081/api/'
+  - url: '93.144.28.153:8081/api/'
```

Comment:
aren't URLs supposed to have a scheme here? see https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#fixed-fields-4